### ISSUE TITLE
ic_menu_black_24dp.xml not present for small screens.

### ISSUE THREAD

**Issue by monday8am** [toggl/mobile#1194]
App: Toggl Time Tracker - 8.1.1 (65) #2
    **Stack Trace**
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.toggl.timer/toggl.joey.ui.activities.MainDrawerActivity}: android.content.res.Resources$NotFoundException: File res/drawable/ic_menu_black_24dp.xml from drawable resource ID #0x7f0200ae
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2436)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2498)
	at android.app.ActivityThread.access$900(ActivityThread.java:179)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1324)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:146)
	at android.app.ActivityThread.main(ActivityThread.java:5641)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1288)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1104)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_menu_black_24dp.xml from drawable resource ID #0x7f0200ae
	at android.content.res.Resources.loadDrawable(Resources.java:3430)
	at android.content.res.Resources.getDrawable(Resources.java:1917)
	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:323)
	at android.support.v7.internal.widget.TintManager.getDrawable(TintManager.java:174)
	at android.support.v7.internal.widget.TintManager.getDrawable(TintManager.java:167)
	at android.support.v7.widget.Toolbar.setNavigationIcon(Toolbar.java:784)
	at md5d0def3047409cb8594a2ecbf8acf6aee.BaseActivity.n_onCreate(Native Method)
	at md5d0def3047409cb8594a2ecbf8acf6aee.BaseActivity.onCreate(BaseActivity.java:30)
	at android.app.Activity.performCreate(Activity.java:5484)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1093)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2400)
	... 11 more
Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #2: invalid drawable tag vector
	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:986)
	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:930)
	at android.content.res.Resources.loadDrawable(Resources.java:3426)
	... 21 more

```
TestFairy session: [3](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/3)
Date: 2016-01-20 15:47:58
User: User-2
Device: samsung - SM-G110B  - 4.4.2 API 19
[TestFairy session 4](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/4): 2016-01-20 15:51:39 - User-2 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 8](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/8): 2016-01-20 15:53:16 - User-2 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 10](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/10): 2016-01-20 15:58:37 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 12](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/12): 2016-01-20 15:59:50 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 15](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/15): 2016-01-20 16:00:17 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 17](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/17): 2016-01-20 16:00:51 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 19](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/19): 2016-01-20 16:06:56 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 21](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/21): 2016-01-20 16:11:13 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 22](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/22): 2016-01-20 16:12:49 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 24](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/24): 2016-01-20 16:14:59 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 28](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/28): 2016-01-20 16:28:45 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 31](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/31): 2016-01-20 16:31:57 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 33](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/33): 2016-01-20 17:22:38 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 36](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/36): 2016-01-20 17:38:05 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 40](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/40): 2016-01-20 18:06:12 - User-3 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 41](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/41): 2016-01-20 19:08:35 - User-4 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 43](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/43): 2016-01-21 03:00:30 - User-4 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 44](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/44): 2016-01-21 03:01:54 - User-5 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 46](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/46): 2016-01-21 03:16:10 - User-5 - samsung - SM-G110B - 4.4.2 API 19
[TestFairy session 48](https://toggl.testfairy.com/projects/1985413-toggltimetracker/builds/2621755/sessions/48): 2016-01-21 05:08:58 - User-5 - samsung - SM-G110B - 4.4.2 API 19

**Comment by monday8am** [comment]
Raygun report https://app.raygun.io/crashreporting/gn9q04/errors/1063293268 has more details about the error

**Comment by monday8am** [comment]
closed by #1370
